### PR TITLE
feat(editor): open at the end of the line

### PR DESCRIPTION
**Pull request by caarlos0**
allows to tell the editor to open the file in the end of the line, should work with and without specifying which line.

### CODE REVIEW

File: editor/editor.go [hunk 1]
```diff
@@ -35,6 +35,15 @@ func OpenAtLine(number uint) Option {
 	}
 }
 
+// OpenAtEndOfLine opens the file at the end of the line.
+func OpenAtEndOfLine(editor, _ string) (args []string, pathInArgs bool) {
```
Comment (aymanbagabas) [inline editor/editor.go:39]: Shouldn't this return an `Option`?

Comment (caarlos0) [reply editor/editor.go:39]: It **is** an option, since there's no arguments, you can simply pass the function

Comment (aymanbagabas) [reply editor/editor.go:39]: oh my bad, i overlooked that part 😅

Comment (aymanbagabas) [reply editor/editor.go:39]: perhaps we could make it explicitly return an `Option`

Comment (caarlos0) [reply editor/editor.go:39]: I mean... there's no reason for it as unlike open at line, it doesn't take any options...

there's no great reason against it either, I think

File: editor/editor.go [hunk 2]
```diff
@@ -17,7 +17,14 @@ const defaultEditor = "nano"
...
 
 // OpenAtLine opens the file at the given line number in supported editors.
+//
+// Deprecated: use Line instead.
 func OpenAtLine(number uint) Option {
+	return Line(number)
+}
+
+// Line opens the file at the given line number in supported editors.
+func Line(number uint) Option {
```
Comment (aymanbagabas) [inline editor/editor.go:27]: Merely a suggestion to make this option clearer 🤔

```suggestion
// LineNr is an option that opens the file at the given line number in supported editors.
func LineNr(number uint) Option {
```

**Review by aymanbagabas — APPROVED** [review]

**Pull request merged**